package.json

```diff
@@ -14,13 +14,17 @@
   "contributors": [
     "Shaangor (https://github.com/Shaangor)",
     "Carl-Hugo Marcotte (https://github.com/Carl-Hugo)",
-    "Layton B (https://github.com/LaytonGB)"
+    "Layton B (https://github.com/LaytonGB)",
+    "Adam D (https://github.com/laquasicinque"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/Carl-Hugo/types-roll20/issues"
   },
   "homepage": "https://github.com/Carl-Hugo/types-roll20#readme",
-  "devDependencies": {},
-  "dependencies": {}
+  "devDependencies": {
+    "@types/underscore": "^1.10.24",
```

Comment:
Why add a dependency on `underscore`'s types?

Reply:
[Roll20s API includes underscore](http://roll20.zendesk.com/hc/en-us/articles/360037256774#API:UtilityFunctions-Underscore.js), although this should probably be a dependency rather than a devDependency, so I'll fix that in a few hours (after work)

Reply:
A second thought on this, why not just leave this one out, and if someone wants to use Underscore, he can just load the types for that library?

Reply:
sure, removed